manifests/params.pp

```diff
@@ -20,8 +20,8 @@
   $config_dir     = '/opt/kafka/config'
   $bin_dir        = '/opt/kafka/bin'
   $log_dir        = '/var/log/kafka'
-  $mirror_url     = 'https://www.apache.org/dyn/closer.cgi?path='
-  $mirror_subpath = "/kafka/${version}"
+  $mirror_url     = 'https://www.apache.org/dyn/closer.lua?action=download&filename='
```

Comment:
this looks like a bugfix that should be made in a separate pull request

Reply:
Yeah, well, in the end you are right, but without it does not work..